src/lib/utils.ts

```diff
@@ -15,3 +16,8 @@ export function formatExpenseDate(date: Date) {
     timeZone: 'UTC',
   })
 }
+
+/** Format category, e.g. for use in AI prompts */
+export function formatCategory(category: Category) {
```

Comment:
I suggest the function name to be more specific, e.g. `formatCategoryForAIPrompt`. Otherwise, one could think this should be used to display a category on the UI.

Reply:
Thanks! I renamed it and removed the TSDoc since I felt it was redundant with the more concise name.